webapp/src/ts/modules/analytics/analytics-targets.component.html

```diff
@@ -48,7 +48,20 @@
             <h2 *ngIf="target.translation_key">{{ target.translation_key | translate }}</h2>
             <h2 *ngIf="!target.translation_key">{{ target.name | translateFrom }}</h2>
             <p *ngIf="target.subtitle_translation_key">
-              {{ target.subtitle_translation_key | translate }}
+
+              <!-- 🔹 Show “Current month” when current filter is selected -->
+  <ng-container *ngIf="reportingPeriodFilter === 'current'; else previousSubtitle">
+    {{ 'targets.this_month.subtitle' | translate }}
+  </ng-container>
+
+  <!-- 🔹 Show “All time” when previous month is selected -->
+  <ng-template #previousSubtitle>
+    {{ 'targets.all_time.subtitle' | translate }}
+    <!-- Inline note: only show when previous period -->
+    
+  </ng-template>
+
+              
```

Comment:
Lets roll this back for now and wait for https://github.com/medic/cht-core/issues/10439 to make any changes to the UI.


```suggestion
              {{ target.subtitle_translation_key | translate }}
```

Reply:
As discussed at the squad meeting this week, we want to go ahead and leave this html as it is for now.  We are continuing to discuss what the best approach will be for the UI changes. These will be implemented in r https://github.com/medic/cht-core/issues/10439.